src/mdacli/libcli.py

```diff
@@ -422,6 +422,8 @@ def create_cli(sub_parser, interface_name, parameters):
                           help=description,
                           default=default,
                           )
+        if group.title == "Mandatory Parameters":
```

Comment:
I would make `"Mandatory Parameters"`a variable inside the function and use that variable in both lines `393` and `425`. There will be many people likely using this lib, and you want to avoid editing line `393` without noticing line `425`.